public/app/features/comments/CommentView.tsx

```diff
@@ -28,8 +28,9 @@ export const CommentView = ({ comments, packetCounter, addComment }: Props) => {
   }, [packetCounter]);
 
   const onUpdateComment = (event: FormEvent<HTMLTextAreaElement>) => {
-    const element = event.target as HTMLInputElement;
-    setComment(element.value);
+    if (event.target instanceof HTMLInputElement) {
+      setComment(event.target.value);
+    }
```

Comment:
I think this change might actually change the implementation as it is today, right?

Reply:
You're right. Maybe something for a separate PR.